src/dynarec/arm64/dynarec_arm64_660f.c

```diff
@@ -308,9 +308,12 @@ uintptr_t dynarec64_660F(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, int n
                     GETGX(q0, 1);
                     GETEX(q1, 0, 0);
                     d0 = fpu_get_scratch(dyn);
-                    MOVIQ_8(d0, 0b10001111);
+                    d1 = fpu_get_scratch(dyn);
+                    MOVIQ_8(d0, 0b1111);
                     VANDQ(d0, d0, q1);  // mask the index
                     VTBLQ1_8(q0, q0, d0);
+                    VSSHRQ_8(d1, q1, 7); // zero mask
+                    VBICQ(q0, q0, d1);

```

Comment:
And why changing this one too?

VTBL1 is supposed to set element to 0 if index is over the source size, and this is done by setting bit 7 of index (so idx>=128, so out of range).